public/manifest.json

```diff
@@ -1,15 +1,15 @@
 {
-  "short_name": "React App",
-  "name": "Create React App Sample",
-  "icons": [
-    {
-      "src": "favicon.ico",
-      "sizes": "64x64 32x32 24x24 16x16",
-      "type": "image/x-icon"
-    }
-  ],
-  "start_url": ".",
-  "display": "standalone",
-  "theme_color": "#000000",
-  "background_color": "#ffffff"
-}
+    "short_name": "Anonygram",
+    "name": "Anonygram-client",
+    "icons": [
+      {
+        "src": "assets/icon.png",
```

Comment:
I realize we have two favicons that we were referencing -- the one in /public and the other in /public/assets. I think the one you wanted it to point to was in src/assets.